integrations/ConvertFlow/browser.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+import { NAME } from "./constants";
+import logger from "../../utils/logUtil";
+import ScriptLoader from "../ScriptLoader";
+
+class ConvertFlow {
+  constructor(config) {
+    this.websiteId = config.websiteId;
+    this.toggleToSendData = config.toggleToSendData;
+    this.eventsList = config.eventsList;
+    this.eventsMappping = config.eventsMappping;
+    this.name = NAME;
+  }
+
+  init() {
+    logger.debug("===In init convertflow===");
+    ScriptLoader(
+      "convertflow-integration",
+      `https://js.convertflow.co/production/websites/${this.websiteId}.js`
+    );
+    if (this.toggleToSendData) {
+      this.trigger();
+    }
```

Comment:
I don't think this is the right place to add this `eventListener`. Let's put this functionality in the isLoaded() function after loading their sdk.

Reply:
Updated